packages/wrangler/src/dev.tsx

```diff
@@ -374,6 +374,14 @@ function useEsbuild(props: {
         ...(jsxFactory && { jsxFactory }),
         ...(jsxFragment && { jsxFragment }),
         external: ["__STATIC_CONTENT_MANIFEST"],
```

Comment:
add `mainFields` here too?

Reply:
I actually think we can remove `mainFields` from the other file because this is the default behavior of esbuild (as long as you're not using `"platform": "node"` setting)

Reply:
Dope, let's remove mainFields there too then

Reply:
done